PulumiUP blog schedule workshops and more (#12688)
pulumi/docs: content/blog/pulumiup-global-cloud-iac-platform-engineering-ai-conference/index.md

    - announcements
    - platform-engineering
    - ai
    - cloud-computing
    - infrastructure-as-code
    - devsecops
    - community

# The social copy used to promote this post on Twitter and Linkedin. These
# properties do not actually create the post and have no effect on the
# generated blog page. They are here strictly for reference.

# Here are some examples of posts we have made in the past for inspiration:
# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
# https://twitter.com/PulumiCorp/status/1763265391042654623
# https://twitter.com/PulumiCorp/status/1762900472489185492
# https://twitter.com/PulumiCorp/status/1755637618631405655

social:
    twitter: Join 5,000+ engineers at PulumiUP 2024! Explore talks on Cloud, DevOps, Platform Engineering & AI/ML. Register now to build your schedule and level up!
    linkedin:

# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
# for details, and please remove these comments before submitting for review.
---

> PulumiUP 2024 is just around the corner, happening on September 18th, and with over 5,000 engineers from all over the world already registered, this is shaping up to be the must-attend event for cloud professionals, platform engineers, and AI/ML enthusiasts alike. From entry-level engineers to tech executives, this event brings together professionals from companies of all sizes to explore the latest innovations and best practices.

Reviewer: throw in the start time because its actually hard to find it on the website

- **"[Using Pulumi to Empower Kubernetes Fleet Management](https://conference.pulumi.com/talks/using-pulumi-to-empower-kubernetes-fleet-management/)"** by Blake Romano (Imagine Learning) will show you how Imagine Learning leverages Pulumi to manage a fleet of Kubernetes clusters and other platform resources quickly and reliably, and how it revolutionized their workflow.
- **"[Streamline Your Infrastructure Deployment with GitOps and Pulumi Operator](https://conference.pulumi.com/talks/streamline-your-infrastructure-deployment-with-gitops-and-pulumi-operator/)"** by Sam Cogan (WTW) will highlight how you can define your infrastructure in Git, and have Pulumi automatically reconcile it into deployed infrastructure.
- **"[Is This a Platform? Platform Engineering Before PMF](https://conference.pulumi.com/talks/is-this-a-platform-platform-engineering-before-pmf/)"** by Jk Jensen (Team Lead, Staff Software Engineer, MystenLabs) will share how they built their service platform around Pulumi to help engineers and researchers move from prototypes to deployed environments, enabling their teams to iterate more quickly and develop an increasing number of systems without additional complexity.

> - **"[Meet Devs Where They Are: Why IaC Must Be Real Code}(https://conference.pulumi.com/talks/meet-devs-where-they-are-why-iac-must-be-real-code/)"** by Jeremy Adams (Head of Ecosystems, Dagger) presents why real code (that teams already use for applications) is essential for both IaC and CI/CD pipelines, and to meet modern DevOps best practices.

Author: ```suggestion
- **"[Meet Devs Where They Are: Why IaC Must Be Real Code](https://conference.pulumi.com/talks/meet-devs-where-they-are-why-iac-must-be-real-code/)"** by Jeremy Adams (Head of Ecosystems, Dagger) presents why real code (that teams already use for applications) is essential for both IaC and CI/CD pipelines, and to meet modern DevOps best practices.
```

## Amazing Partners and Sponsors

Thank you to our amazing partners for supporting PulumiUP!

- **[AWS](https://aws.amazon.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?
- **[Google Cloud](https://cloud.google.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?
- **[New Relic](https://developer.newrelic.com/pulumi/get-started-pulumi/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?

## Build Your Schedule Now

With so many compelling topics across AI/ML, Platform Engineering, DevOps, and Cloud and IaC, now is the time to [build your PulumiUP schedule](https://conference.pulumi.com/schedule/?utm_source=PulumiUP&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP). Choose the sessions that align with your goals and interests to get the most out of this unique virtual event. Whether you're looking to sharpen your skills, discover the latest innovations, or connect with industry leaders, PulumiUP has something for everyone.

> Haven’t registered yet? It’s not too late! Join 5,000 engineers worldwide in this groundbreaking event and gain the insights you need to drive innovation at your company. [Register now->](https://www.pulumi.com/pulumi-up/)

Reviewer: should we mention that if they miss it, their will be recordings sent out? We can ask them to subscribe to the newsletter to get links to the recordings, that way even if they don't attend, we are still getting them into the database

Author: Good point. We actually tell them to register to get access to the recordings - it is definitely not timezone-friendly for many.

{{< figure alt="Panelists of Infrastructure as Code - Can we do better?" src="./pulumiup_2024-infrastructure-as-code.png" caption="Panel: Infrastructure as Code - Can we do better?" width=100% >}}

## Workshops and Hands-On Learning

Want to get hands-on experience? Our live workshops are designed to deepen your understanding of trending topics, Pulumi, and its applications.

- **"[Managing Team Secrets with 1Password & Pulumi ESC](https://www.pulumi.com/resources/managing-team-secrets-with-1password-pulumi-esc/)"** on September 25, led by Phil Johnston (1Password) and Diana Esteves (Pulumi).
- **"[Securing IaC Pipelines in Regulated Industries](https://www.pulumi.com/resources/securing-iac-pipelines-in-regulated-industries/)"** on September 26, led by Josh Kodroff (Pulumi) and Marina Novikova (AWS).

> - **"[Observability as Code for AI Apps with New Relic and Pulumi](https://www.pulumi.com/resources/observability-as-code-for-ai-apps-new-relic/)"** on October 30, led by Diana Esteves (Pulumi) and Harry Kimpel (New Relic).

Reviewer 2: ```suggestion
- **"[Deploy Tailscale infrastructure in AWS with Pulumi](https://www.pulumi.com/resources/deploy-tailscale-infrastructure-with-pulumi/)"** on October 15, led by Diana Esteves (Pulumi) and Lee Briggs (Tailscale).
- **"[Observability as Code for AI Apps with New Relic and Pulumi](https://www.pulumi.com/resources/observability-as-code-for-ai-apps-new-relic/)"** on October 30, led by Diana Esteves (Pulumi) and Harry Kimpel (New Relic).
```

## Amazing Partners and Sponsors

> Thank you to our amazing partners for supporting PulumiUP!

Reviewer 2: ```suggestion
Thank you to our amazing customers, sponsors and partners for supporting PulumiUP!
```

> - **[AWS](https://aws.amazon.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?

Reviewer 2: ```suggestion
- **[AWS](https://aws.amazon.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** The world's most comprehensive and broadly adopted cloud, offering over 200 fully featured services from data centers globally.
```

> - **[Google Cloud](https://cloud.google.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?

Reviewer 2: ```suggestion
- **[Google Cloud](https://cloud.google.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Google Cloud helps developers build quickly, securely, and cost effectively with the next generation of modern infrastructure designed to meet specific workload and industry needS.```

> - **[New Relic](https://developer.newrelic.com/pulumi/get-started-pulumi/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** yadda yadda?

Reviewer 2: ```suggestion
- **[New Relic](https://developer.newrelic.com/pulumi/get-started-pulumi/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Data for engineers to monitor, debug, and improve their entire stack.
```

> 

Reviewer 2: ```suggestion
- **[Docker](https://docker.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Accelerate how you build, share, and run applications.
- **[1Password](https://1password.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** 1Password streamlines how you manage SSH keys, API tokens, and other infrastructure secrets throughout the entire software development life cycle.
- **[Tailscale](https://tailscale.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Tailscale makes creating software-defined networks easy: securely connecting users, services, and devices.
- **[Pinecone](https://pinecone.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Build remarkable GenAI applications fast, with lower cost, better performance, and greater ease of use at any scale.
- **[Wiz](https://wiz.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Secure everything you build and run in the cloud.
- **[Dagger](https://dagger.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Powerful, programmable open source CI/CD engine that runs your pipelines in containers.
- **[Oso](https://osohq.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Authorization as a service.
```

---
title: "PulumiUP 2024: Dive Into the Future of Cloud, Platform Engineering, and AI/ML"
allow_long_title: true

# The date represents the post's publish date, and by default corresponds with
# the date and time this file was generated. Dates are used for display and
# ordering purposes only; they have no effect on whether or when a post is
# published. To influence the ordering of posts published on the same date, use
# the time portion of the date value; posts are sorted in descending order by
# date/time.

> date: 2024-09-11T09:04:40Z

Author: ```suggestion
date: 2024-09-11T07:04:40Z
```

    - announcements
    - platform-engineering
    - ai
    - cloud-computing
    - infrastructure-as-code
    - devsecops
    - community

# The social copy used to promote this post on Twitter and Linkedin. These
# properties do not actually create the post and have no effect on the
# generated blog page. They are here strictly for reference.

# Here are some examples of posts we have made in the past for inspiration:
# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
# https://twitter.com/PulumiCorp/status/1763265391042654623
# https://twitter.com/PulumiCorp/status/1762900472489185492
# https://twitter.com/PulumiCorp/status/1755637618631405655

social:
    twitter: Join 5,000+ engineers at PulumiUP 2024! Explore talks on Cloud, DevOps, Platform Engineering & AI/ML. Register now to build your schedule and level up!
    linkedin:

# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
# for details, and please remove these comments before submitting for review.
---

> PulumiUP 2024 is just around the corner! It will be held on September 18th, starting at 8 AM PT | 15:00 UTC +0, and with over 5,500 engineers from all over the world already registered, this is shaping up to be the must-attend event for cloud professionals, platform engineers, and AI/ML enthusiasts alike. From entry-level engineers to tech executives, this event brings together professionals from companies of all sizes to explore the latest innovations and best practices.

Reviewer 3: > ... the latest innovations and best practices.
The latest innovations and best practices in what? Maybe cloud, platform engineering, and AI/ML?

> If you haven’t registered yet, now’s the time! Start building your schedule today, and join us for insightful sessions across our topic tracks: **Cloud and IaC**, **Platform Engineering & DevOps**, and **AI/ML**.

Author: The actual name is Cloud and IaC track so I will keep it as is.

> If you haven’t registered yet, now’s the time! [Start building your schedule today](https://conference.pulumi.com/schedule/?utm_source=PulumiUP&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP), select the talks you want to watch live and on-demand and add them to your schedule. All talks and panels will be immediately available on demand.

Author: ```suggestion
If you haven’t registered yet, now’s the time! [Start building your schedule today](https://conference.pulumi.com/schedule/?utm_source=PulumiUP&utm_medium=web&utm_campaign=FY2025Q1_Event_PulumiUP), select the talks you want to watch live and on-demand and add them to your schedule.
```

- **[Wiz](https://wiz.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Secure everything you build and run in the cloud.
- **[Honeycomb](https://honeycomb.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Observability that helps solve problems you couldn’t before.
- **[Mysten Labs](https://mystenlabs.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Building critical infrastructure to enable a more decentralized internet.
- **[Imagine Learning](https://imaginelearning.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Empowering educators to inspire breakthrough moments in every student’s unique learning journey with digital-first, K–12 education solutions.
- **[Dagger](https://dagger.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Powerful, programmable open source CI/CD engine that runs your pipelines in containers.
- **[Oso](https://osohq.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP)** Authorization as a service.

> ## Build Your Schedule Now

Author: The same call to action is in the intro, but I will also add the link to it.
